ethexe/network/src/db_sync/ongoing.rs

```diff
@@ -113,25 +152,53 @@ impl OngoingRequest {
         response
     }
 
+    fn inner_complete(
+        mut self,
+        peer: PeerId,
+        response: Response,
+    ) -> Result<(RequestId, Response), Self> {
+        if let Some(new_request) = self.request.difference(&response) {
```

Comment:
What is `validate_response` inside `difference` impl?

Reply:
Artifact after renamings